lib/util/exec/index.spec.ts

```
@@ -189,15 +201,15 @@ describe('util/exec/index', () => {
         outCmd: [
           dockerPullCmd,
           dockerRemoveCmd,
-          `docker run --rm --name=${name} --label=renovate_child ${defaultVolumes} ${defaultCwd} ${fullImage} bash -l -c "${inCmd}"`,
+          `docker run --rm --name=${name} --label=renovate_child ${defaultVolumes} -e BUILDPACK_CACHE_DIR ${defaultCwd} ${fullImage} bash -l -c "${inCmd}"`,
```

I would like to see this to be tested as new separate cases, instead of modifying parameters for existing ones

---

Not possible, as it's now always added for `binarySource=docker`

---

should we add a admin config to enable the cache?

---

Which minimum version of buildpack will this require? I guess it will degrade gracefully though if newer Renovate is run within older buildpack?

---

not sure, but the variables is only used by V2 tools. it doesn't matter. it will silently ignored by older buildpack versions